tool/tctl/common/plugin/awsic.go

```diff
@@ -198,6 +219,13 @@ func (p *PluginsCommand) initInstallAWSIC(parent *kingpin.CmdClause) {
 		StringsVar(&p.install.awsIC.accountNameFilters)
 	cmd.Flag("account-id", "Add AWS Account to account import list by ID. All AWS accounts will be imported if no items are added to account import list.").
 		StringsVar(&p.install.awsIC.accountIDFilters)
+
```

Comment:
we are not supporting `exclude-group-id`?

Reply:
Right the AWS ID groups  are not supported in `include` or `exclude` filter. The AWS groups needs. in AWS IC the group needs to have uniq name so group name  == group id.